crates/anvil/src/eth/backend/executor.rs

```diff
@@ -211,8 +211,9 @@ impl<DB: Db + ?Sized, V: TransactionValidator> TransactionExecutor<'_, DB, V> {
             transactions.push(transaction.pending_transaction.transaction.clone());
         }
 
-        let receipts_root =
-            trie::ordered_trie_root(receipts.iter().map(Encodable2718::encoded_2718));
+        let receipts_root = root::ordered_trie_root(
+            &receipts.iter().map(Encodable2718::encoded_2718).collect::<Vec<_>>(),
+        );
```

Comment:
and here we can use:

https://github.com/alloy-rs/alloy/blob/833f9a22ca7c30079d64554d435561877c15267a/crates/consensus/src/proofs.rs#L49-L49